File: Python/Conditional Statement/tax.py
income = float(input("Enter your income: "))

tax = 0

if income <= 300000:
    tax = 0
elif income <= 600000:
    tax = (income - 300000) * 0.05
elif income <= 900000:
    tax = (300000 * 0.05) + (income - 600000) * 0.10
elif income <= 1200000:
    tax = (300000 * 0.05) + (300000 * 0.10) + (income - 900000) * 0.15
elif income <= 1500000:
    tax = (300000 * 0.05) + (300000 * 0.10) + (300000 * 0.15) + (income - 1200000) * 0.20
else:
    tax = (300000 * 0.05) + (300000 * 0.10) + (300000 * 0.15) + (300000 * 0.20) + (income - 1500000) * 0.30

print(tax)

File: C/Conditions/tax.c
# include <stdio.h>

int main () {
	
	int salary;
	float tax, aftertax;
	
	printf("Please enter your salary:");
	scanf("%d",&salary);
	
	if(salary<=2000){
		printf("No Tax Deduction");
	} else if (salary<=4000 && salary>2000 ){
		tax = salary*0.03;
	} else if (salary<=5000 && salary>4000){
		tax = salary*0.05;
	} else if (salary>5000){
		tax = salary*0.08;
	}
	
	aftertax = salary-tax;
	
	printf("Your Tax deduction is:%f\n",tax);
	printf("Your Salary after tax deduction is:%f",aftertax);
	
	return 0;
}
